src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper/help.html

```diff
@@ -1,4 +1,6 @@
 <dl>
+    <dt><code>getBuildCauses</code></dt><dd>Returns a JSON array of build causes for the current build</dd>
+    <dt><code>getBuildCauses(Class&lt;Cause&gt;)</code></dt><dd>Returns a <code>JSON array</code> of build causes of type <code>Cause</code> for the current build, or an empty <code>JSON array</code> if no causes of the specified type apply to the current build</code></dd>
```

Comment:
iff we are thinking of changing this API can you mark it as BETA or EXPERIMENTAL.